From 37fbb4d76ba4a06099e1e56cc7c3c96c985f2afa Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Mon, 29 Sep 2025 19:38:21 +0200
Subject: [PATCH] board: solidrun: imx8mm_som: move environment to fit within
 emmc boot-part

Move environment to last erase block (512k), just before 4MiB size limit
of eMMC boot partitions.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 configs/imx8mm_sr_som_defconfig | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/configs/imx8mm_sr_som_defconfig b/configs/imx8mm_sr_som_defconfig
index d394c37d..96bc597c 100644
--- a/configs/imx8mm_sr_som_defconfig
+++ b/configs/imx8mm_sr_som_defconfig
@@ -5,8 +5,9 @@ CONFIG_SYS_MALLOC_LEN=0x2000000
 CONFIG_SPL_GPIO=y
 CONFIG_SPL_LIBCOMMON_SUPPORT=y
 CONFIG_SPL_LIBGENERIC_SUPPORT=y
-CONFIG_ENV_SIZE=0x4000
-CONFIG_ENV_OFFSET=0x700000
+CONFIG_ENV_SIZE=0x80000
+CONFIG_ENV_OFFSET=0x380000
+CONFIG_ENV_SECT_SIZE=0x10000
 CONFIG_SYS_MEMTEST_START=0x40000000
 CONFIG_SYS_MEMTEST_END=0x80000000
 CONFIG_SYS_I2C_MXC_I2C1=y
-- 
2.51.0

